### PR TITLE
Wrote code for popup menu

### PR DESCRIPTION
**Pull request by Amanulla**
### Popup Modal
By doing this activity, I learned how to make popup modal windows and style them to look good. And I learn how to set Attribute and how to get Attribute in javascript.
- I wrote code for Modals for mobile and desktop versions.
- Added a few lines of javascript to popup the modal window.
- Added some style to the particular Modals to look better.

### CODE REVIEW

**Review by tresorsawasawa — COMMENTED** [review]
# STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Amanulla 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that requires [pair-programming](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20should%20implement%20the%20above,and%20submit%20this%20activity%20individually.) but I noticed that the current PR has only one contributor. Kindly make sure your coding partner has some commits in your pull request before requesting the next review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

---
Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

---
_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by zieeco — COMMENTED** [review]
# STATUS: INVALID!!!  :x: :x: :x: 

Hi @Amanulla ,




While you made a great effort in this project, unfortunately, I cannot proceed to review your code, please make sure you implement the following changes before you resubmit for review. 

## Required Changes ♻️
_Check the comments below._

- [ ] As it is a requirement for a project that students should not edit the linter file as you've done, please kindly remove all the linters files you have now and copy that of the [microverse html-css-js](https://github.com/microverseinc/linters-config/tree/master/html-css-js), delete the node_modules file and install them using npm. that should update all the linters.

- [ ] As it is a requirement for this project, please, kindly make sure there are no linter errors.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Invalid Code Review Request **Does not count** into the code reviews limit._
# **INVALID PROJECT**

**Review by zieeco — COMMENTED** [review]
# STATUS: INVALID!!!  :x: :x: :x: 

Hi @Amanulla ,




While you made a great effort in this project, unfortunately, I cannot proceed to review your code, please make sure you implement the following changes before you resubmit for review. 

## Required Changes ♻️
_Check the comments below._

- [ ] As it is a requirement for a project that students should not edit the linter file as you've done, please kindly remove all the linters files you have now and copy that of the [microverse html-css-js](https://github.com/microverseinc/linters-config/tree/master/html-css-js), delete the node_modules file and install them using npm. that should update all the linters.

- [ ] As it is a requirement for this project, please, kindly make sure there are no linter errors.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Invalid Code Review Request **Does not count** into the code reviews limit._
# **INVALID PROJECT**

File: index.html [hunk 1]
```diff
@@ -75,9 +78,62 @@ <h2 class="tonic1">Tonic</h2> <br>
               <li class="back4">css</li>
               <li class="back4">javascript</li>
             </ul><br>
-            <button type="button" class="button button4">See Project</button>
+            <button type="button" class="button button4 myBtn">See Project</button>
+          </div>
+        </div>
+
+        <!-- --------------------------------------------First Modal--------------------------------------------- -->
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] Please remove all the comments that look similar to this and run `npx hint .` on your terminal. This should fix the error. HTML comments should be written in this format `<!-- Comment -->`

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]

File: app1.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  btn[i].onclick = function () {
+    const ElementIndex = this.getAttribute('data-index', i);
+    modal[ElementIndex].style.display = 'block';
+  };
+
+  span[i].onclick = function () {
+    const ElementIndex = this.getAttribute('data-index', i);
+    modal[ElementIndex].style.display = 'none';
+  };
+}
```
Comment (Lucas-Masaba) [inline app1.js]: - [ ] According to the instructions, you are supposed to use a javascript object to store information for all projects as seen here. Please fix that 👍  
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Amanulla  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- For some reason, when I run your project live, it does not run on my browser and so I cannot check the popup functionality.

**Review by ishimwezachee — APPROVED** [review]
Hi @Amanulla and @Lucas-Masaba , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**